pkg/antctl/raw/resourceexport/command.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourceexport
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+var Command *cobra.Command
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "resourceexport",
+		Aliases: []string{"resourceexports"},
+		Short:   "Print Multi-cluster resourceexports",
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 0 {
+				_ = runE(cmd, args[0])
+			} else {
+				_ = runE(cmd, "")
+			}
+		},
+	}
+
+}
+
+func runE(cmd *cobra.Command, ClusterID string) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	kubeconfig.GroupVersion = &schema.GroupVersion{Group: "", Version: ""}
+
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+	k8sClient, err := raw.AddSchemeToClientCfg(kubeconfig)
+	if err != nil {
+		return err
+	}
+
+	res := &multiclusterv1alpha1.ResourceExportList{}
+
+	err = k8sClient.List(context.TODO(), res, &client.ListOptions{Namespace: metav1.NamespaceAll})
+
+	if err != nil {
+		return err
+	}
+
+	var resExports []multiclusterv1alpha1.ResourceExport
+
+	if ClusterID != "" {
+		for _, r := range res.Items {
+			if r.Labels["sourceClusterID"] == ClusterID {
+				resExports = append(resExports, r)
+			}
+		}
+	} else {
+		resExports = res.Items
+	}
+
+	fmt.Println(output(resExports))
+
+	return nil
+}
+
+func output(resExports []multiclusterv1alpha1.ResourceExport) string {
+	var output strings.Builder
+	formatter := "%-50s%-50s%-50s\n"
+	output.Write([]byte(fmt.Sprintf(formatter, "CLUSTERID", "NAME", "NAMESPACE")))
```

Comment:
the sequence is wrong, it's inconsistent with output, it should be `"NAMESPACE", "NAME"`.

Reply:
I would suggest to support -o for all commands.